src/runtime/node/perf_hooks/internal/performance.ts

```diff
@@ -30,6 +30,7 @@ const nodeTiming = {
   loopStart: 63.262_040_972_709_656,
   loopExit: -1,
   idleTime: 305_360.555_328,
+  uvMetricsInfo: { loopCount: 142, events: 137, eventsWaiting: 97 },
```

Comment:
```suggestion
  uvMetricsInfo: { loopCount: 0, events: 0, eventsWaiting: 970},
```

(i just noticed, other values should be zeroed as well they seem random snapshot...

Reply:
I used a snapshot here because:
```
// grabbed from Node.js v22.3.0 using:
//   performance.nodeTiming
```

But it makes sense to zero out, I'll update all the values